Snippets/end.Checks.groovy

```diff
@@ -467,10 +467,11 @@ def existsCheck(String codeMissing, String sheet, String field, boolean isWarnin
     generalCheck(null, codeMissing, sheet, field, null, exists(), null, isWarning)
 }
 
-def urlCheck(String codeInvalid, String codeMissing, String sheet, String field, String mustStartWith = null, boolean isWarning = false) {
-    generalCheck(codeInvalid, codeMissing, sheet, field, 'is not a valid URL',
+def urlCheck(String codeInvalid, String codeMissing, String sheet, String field, String mustStartWith = null, boolean isWarning = false, long maxLength = _project.vars.MAX_URL_LENGTH as long) {
+    (generalCheck(codeInvalid, codeMissing, sheet, field, 'is not a valid URL',
```

Comment:
The outer brackets around the whole statement can be removed

Reply:
I removed them.